pkg/apis/batch/validation/validation.go

```diff
@@ -513,6 +525,13 @@ func validateTimeZone(timeZone *string, fldPath *field.Path) field.ErrorList {
 		return allErrs
 	}
 
+	for _, part := range strings.Split(*timeZone, "/") {
+		if part == "." || part == ".." || strings.HasPrefix(part, "-") || !validTimeZoneCharacters.MatchString(part) {
+			allErrs = append(allErrs, field.Invalid(fldPath, timeZone, fmt.Sprintf("unknown time zone %s", *timeZone)))
```

Comment:
Here we already split it into parts, should we mention which part is an unknown timezone in the error message?

Reply:
I don't think there's much value in that granularity, they should check the overall value for validity